python/ray/tests/test_state_api.py

```diff
@@ -2671,7 +2692,7 @@ def test_get_id_not_found(shutdown_only):
 
 
 if __name__ == "__main__":
-    import os
+    import os  # noqa: F811
```

Comment:
why are we adding special rules for linters here?

Reply:
This is no longer required because we are using monkeypatch to set the env var